### DISCUSSION TITLE
How to change res to 4k

### DISCUSSION THREAD

**Discussion by secretkl** [mov-cli/mov-cli#296] [Q&A] [answered]
^Title explains it.

**Comment by THEGOLDENPRO** [comment, marked as answer]
The resolution can only be changed to 4k if the provider the scraper is scraping from supports it. 

The only scrapers I'm certain that support it is yt-dlp from ``mov-cli-youtube``. The highest resolution is usually chosen by default but that choice is up to the scraper, to force the resolution, use the [quality config key](https://github.com/mov-cli/mov-cli/wiki/Configuration#quality).
